RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEGeometric.cc

```diff
@@ -0,0 +1,28 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEGeometric.h"
+#include "SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerFwd.h"
```

Comment:
as I commented to @suchandradutta - the code you are pulling in via this header has to move - we don't want Reco depending on SIM. The best solution in this case is not to use the (to me, strange) typedef of Phase2TrackerTopology and Phase2TrackerDetUnit

but then why do we need a phase 2 specific version of this class?